soc/arm/infineon_cat1/psoc6/Kconfig.defconfig.soc.psoc6_02

```diff
@@ -43,6 +43,7 @@ config SOC
 	default "CY8C614ALQI_S2F02" if SOC_CY8C614ALQI_S2F02
 	default "CY8C6148LQI_S2F42" if SOC_CY8C6148LQI_S2F42
 	default "CY8C6148LQI_S2F02" if SOC_CY8C6148LQI_S2F02
+	default "CY8C6244LQI_S4D92" if SOC_CY8C6244LQI_S4D92
```

Comment:
remove this, you added it in soc/arm/infineon_cat1/psoc6/Kconfig.defconfig.soc.psoc6_04